Docs Enhancements To CLI Section (#689)
spiceai/docs: spiceaidocs/docs/cli/reference/add.md

pagination_next: null
---

> Adds a Spicepod to the project from a given source.

Reviewer: ```suggestion
Adds a Spicepod to the project from a given package.
```

### Usage

```shell

> spice add [source] [flags]

Reviewer: ```suggestion
spice add [package] [flags]
```

```

> - `source`: Location of the Spicepod.

Reviewer: ```suggestion
- `package`: The package to add (i.e. `spiceai/quickstart`).
```

#### Flags

- `-h`, `--help` Print this help message

### Examples

> Adding a Spicepod from a source currently using Spice (like `spiceai/quickstart`):

Reviewer: ```suggestion
Adding a Spicepod from Spicerack using Spice (like `spiceai/quickstart`):
```
